.github/workflows/check-semgrep-pro.jsonnet

```diff
@@ -95,7 +95,7 @@ local check_compile_semgrep_pro_job = {
       run: |||
         cd ../semgrep-proprietary
         eval $(opam env)
-        make
+        make all test
```

Comment:
because the name: is "Compile semgrep-pro", maybe you want to make a separate { name: 'Test semgrep-pro'  ...} with just make test

Reply:
Seems like I would have to copy-paste a large chunk of code, are you sure you want a different job rather than renaming the job ?

Reply:
Well, maybe you meant a separate step ? I'll do that.

Reply:
Did that, let me know if you wanted something else!